src/main/java/net/neoforged/neoforge/server/command/ConfigCommand.java

```diff
@@ -34,11 +36,17 @@ private static int showFile(final CommandContext<CommandSourceStack> context) {
             final String configFileName = ConfigTracker.INSTANCE.getConfigFileName(modId, type);
             if (configFileName != null) {
                 File f = new File(configFileName);
+                MutableComponent fileComponent = Component.literal(f.getName()).withStyle(ChatFormatting.UNDERLINE);
+
+                // Click action not allow on dedicated servers as client cannot click link to a server's file path.
+                if (!FMLLoader.getDist().isDedicatedServer()) {
```

Comment:
This will pose a problem when sending to clients in MP worlds. You should check that the player isn't null, and that its connection is in-memory.

Reply:
Do you have an example of said checks?

Also, wouldn't using this check be proper for your scenario anyway? Since the server is the one with the common and server configs. Only the client would have the client config.

So not sending the click action would be correct as the path given to the client would be the server's common and server config paths. The client config would probably be better as a client command but that would be a bit out of scope of this PR tbh since that may be tricky to setup so only the client arg ends up as a client command while the other args remain serversided.

Reply:
Tested on dedicated server, the server and common configs are working and sending correct names to client without the click actions. The client config is crashing tho since it doesn't exist on server.
![image](https://github.com/neoforged/NeoForge/assets/40846040/44f8a69d-2678-49c6-8165-5e0c9bff827e)

How would you like to proceed? Should this client config arg be split off into it's own PR to figure out how to handle that right? Or is there an example that I can easily use for splitting off an command's one argument to be a client argument? Or would that be it's own command? `/config showfileclient neoforge CLIENT` vs current `/config showfile neoforge CLIENT`

Reply:
Fixed now properly